packages/wonder-blocks-core/src/hooks/__tests__/use-is-mounted.test.ts

```diff
@@ -10,7 +12,6 @@ describe("useIsMounted", () => {
         const {result} = renderHook(useIsMounted);
 
         // Assert
-        // @ts-expect-error [FEI-5019] - TS2339 - Property 'toBeTrue' does not exist on type 'JestMatchers<boolean>'.
```

Comment:
**note:** Importing jest-extended in any file during the typecheck caused it to be available for ALL files, so these suppressions were no longer needed. However, I added the `jest-extended` import anyway since that will ensure it works for folks editing files too where the global typechecking isn't going to be running.